Source/WebCore/css/CSSRayValue.cpp

```diff
@@ -35,10 +35,10 @@ String CSSRayValue::customCSSText() const
 {
     StringBuilder builder;
 
-    builder.append("ray(");
-    builder.append(m_angle->cssText());
-    builder.append(" ");
-    builder.append(m_size->cssText());
+    builder.append("ray(", m_angle->cssText());
+
+    if (m_size->valueID() != CSSValueClosestSide)
+        builder.append(" ", m_size->cssText());
```

Comment:
This should use `‘ ‘` since appending a character is more efficient than appending a string even when the string is known at compile time in our current implementation. Also it _could_ be refactored to use a single call to `makeString` instead. That is not particularly challenging. but note that we might switch back to `StringBuilder` if we can start fixing these to not use intermediate strings but appending the text of sub-values with append functions